Automattic/Simplecons
# Add Success Icon

### Fix

Add "success" icon.

<img width="386" alt="Screenshot 2021-01-26 at 10 35 05" src="https://user-images.githubusercontent.com/36532468/105834266-2ab91980-5fc2-11eb-8ed8-728135784715.png">


### To Test

- Verify the icon appears as intended on a 24px grid.
- Verify the SVG code is consistent with other icons.
